subtract.json

```
@@ -1,79 +1,64 @@
 {
     "id": "subtract",
-    "summary": "Subtraction of a sequence of numbers",
-    "description": "Takes the first element of a sequential array of numbers and subtracts all other elements from it.\n\nThe computations should follow [IEEE Standard 754](https://ieeexplore.ieee.org/document/4610935) whenever the processing environment supports it. Otherwise an exception must the thrown for incomputable results.\n\nBy default no-data values are ignored. Setting `ignore_nodata` to `false` considers no-data values so that `null` is returned if any element is such a value.",
+    "summary": "Subtraction of two numbers",
+    "description": "Subtracts the second value from the first value and returns the computed result.\n\nNo-data values are taken into account so that `null` is returned if any element is such a value.\n\nThe computations follow [IEEE Standard 754](https://ieeexplore.ieee.org/document/8766229) whenever the processing environment supports it.",
```

I suggest to remove position of parameters from the description

```suggestion
    "description": "Subtracts `y` from `x` and returns the computed result (`x - y`).\n\nNo-data values are taken into account so that `null` is returned if any element is such a value.\n\nThe computations follow [IEEE Standard 754](https://ieeexplore.ieee.org/document/8766229) whenever the processing environment supports it.",
```